app/models/journey.rb

```diff
@@ -1,21 +1,10 @@
-# require 'aws-record'
+# frozen_string_literal: true
 
-# class JourneyNoIdError < StandardError; end
-# class Journey < Sequel::Model
-  # def initialize(attrs)
-  #   super(attrs)
-  # end
-  # include Aws::Record
-  # set_table_name ENV["JOURNEY_TABLE_NAME"]
+class Journey < ActiveRecord::Base
+  validates :annual_miles, presence: true, numericality: true
+  validates :monthly_miles, presence: true, numericality: true
+  validates :weekly_miles, presence: true, numericality: true
+  validates :title, presence: true, uniqueness: true
 
-  # string_attr  :title,       hash_key: true
-  # integer_attr :target_miles
-  # string_attr  :graphic_url
-
-  # def self.new_journey(*data)
-  #   binding.pry
-  #   journey = new(*data)
-  #   journey.save!
-  #   journey
-  # end
-# end
+  MILEAGE_CONVERSION_FACTOR = 0.01
```

Comment:
YAGNI...yet. I bet we'll end up adding this to other models that have a mileage and we can extract it into a module at that point if we really want. The indirection of a module vs the repetition is a tradeoff, I think I'd rather be repetitive and clear. We'll cross that bridge when we come to it.